conn_test.go

```diff
@@ -528,11 +550,22 @@ func TestPSK(t *testing.T) {
 					}
 					return []byte{0xAB, 0xC1, 0x23}, nil
 				},
-				PSKIdentityHint: test.ServerIdentity,
-				CipherSuites:    test.CipherSuites,
+				PSKIdentityHint:  test.ServerIdentity,
+				CipherSuites:     test.CipherSuites,
+				VerifyConnection: test.ServerVerifyConnection,
 			}
 
 			server, err := testServer(ctx, cb, config, false)
+			if test.WantFail {
+				if err == nil {
```

Comment:
It would be better to check which error was returned.

Reply:
fixed